### PR TITLE
[GH-852] Configurator: Migrate crates config

### PR DESCRIPTION
**Pull request by agustinesco**
## Motivation

Keep migrating field from config.json to configurator app

Part of https://github.com/lambdaclass/mirra_backend/issues/852

## Summary of changes

- Added crates list to map_configuration schema
- Added edit and update for these fields in configurator app
- Used map config instead of fields from json in game updater to spawn crates

## How to test it?

Start a match and everything should keep working as before

## Checklist
- [x] Tested the changes locally.
- [x] Reviewed the changes on GitHub, line by line.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

File: apps/arena/lib/arena/configuration.ex [hunk 1]
```diff
@@ -184,6 +185,15 @@ defmodule Arena.Configuration do
     }
   end
 
+  defp parse_crate(crate) do
```
Comment (tvillegas98) [inline apps/arena/lib/arena/configuration.ex:188]: It seems parse_pool, parse_bush, and parse crate do the same thing, we can have just one function for them 🤔 . What do you think?

**Review by tvillegas98 — CHANGES_REQUESTED** [review]
Nice work overall! a left a comment about an opportunity to refactor

File: apps/arena/lib/arena/configuration.ex (hunk 1 above)
Comment (agustinesco) [reply apps/arena/lib/arena/configuration.ex]: Sounds good! 🪨

**Review by tvillegas98 — DISMISSED** [review]
Nice work!

**Review by AminArria — APPROVED** [review]

**Review by AminArria — APPROVED** [review]

**Review by AminArria — APPROVED** [review]

**Review by tvillegas98 — APPROVED** [review]

**Pull request merged**